Add renew functionality for client certificates.

eiraelr opened the pull request:
The code does the following:
Generates a new CSR bases on current certification information.
Asks the server to revoke the current certificate.
Sends the CSR to the tsa server.
Replaces the certificate and keys on disk.

eiraelr committed:
Add renew functionality for client certificates.

The code does the following:
Generates a new CSR bases on current certification information.
Asks the server to revoke the current certificate.
Sends the CSR to the tsa server.
Replaces the certificate and keys on disk.

dantremblay commented:
Hi @juliengk , can you please give your comments on this PR?  This is a functionality we currently need in our organization.  Appreciate your feedback.  Thank you.

eiraelr committed:
Add flag to set engine certificate duration.

eiraelr committed:
add newer go version and remove static linking

Build against 1.19.3 and remove static linking of twic as it get SIGSEGV
on newer systems when static linking is used.

eiraelr committed:
don't build centos 6 packages

Not supported any more, noone should use it.
